static/src/javascripts/__flow__/types/acquisitions.js

```diff
@@ -12,6 +12,7 @@ declare type AcquisitionsEpicTemplateCopy = {
 };
 
 declare type EngagementBannerTemplateParams = {
+    leadSentence: string,
```

Comment:
![image](https://user-images.githubusercontent.com/7304387/52805145-e0149000-307d-11e9-8797-cbc7ae617517.png)

Reply:
we haven't yet added in the main text above, as we are not sure if it will be an SVG list or a string (re @paperboyo @GHaberis @ionamckendrick  discussion about font download times )